assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -165,19 +165,14 @@ function BannerNotification( {
 		}, 350 );
 	}
 
-	const isNavigatingToCTALink = useSelect( ( select ) =>
-		select( CORE_LOCATION ).isNavigatingTo( ctaLink || '' )
+	const isNavigating = useSelect( ( select ) =>
+		select( CORE_LOCATION ).isNavigating()
```

Comment:
Check `isNavigating` here as `onCTAClick` may call `navigateTo` with a different address, in which case we still want to keep the spinner visible.